README.md

```diff
@@ -1,5 +1,4 @@
-pvlib-python
-============
+<img src="docs/sphinx/source/_images/pvlib_logo_horiz.png" width="600" height="260">
```

Comment:
@cwhanse sorry to reopen this, but I think you should remove the `height="260"` from the `<img>` tag because it's rendering weird in my phone.
![image](https://user-images.githubusercontent.com/1385621/55927180-f4ca5b80-5bc8-11e9-94d6-cbf3a09061e7.png)
If I remove the `height="260"` then it renders fine:
![image](https://user-images.githubusercontent.com/1385621/55927200-11ff2a00-5bc9-11e9-9e45-aab739b0acf7.png)

Reply:
Note the Sphinx/Readthedocs html only uses `width="600"` and no height:
![image](https://user-images.githubusercontent.com/1385621/55927262-4e328a80-5bc9-11e9-8098-fd3105b662b6.png)
and renders fine on responsive devices:
![image](https://user-images.githubusercontent.com/1385621/55927298-6acec280-5bc9-11e9-8217-05a84592ef08.png)

Reply:
I did check when I clicked approve, and it seemed fine then, so not sure what happend :(